src/extension/utils.js

```diff
@@ -566,8 +566,8 @@ export async function getProjectEnv({
     if (project) {
       env.project = project;
     }
-    if (contentSourceUrl) {
-      env.mountpoints = [contentSourceUrl];
+    if (mountpoints) {
+      env.mountpoints = mountpoints;
```

Comment:
Keep it as an array

Reply:
fixed in [e5b214b](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/e5b214b202d1e462982b19e00ef5de3a2696d022)